apps/meteor/client/components/UserInfo/UserInfoAction.tsx

```diff
@@ -1,16 +1,23 @@
-import { Button } from '@rocket.chat/fuselage';
+import { Button, IconButton } from '@rocket.chat/fuselage';
 import type { Keys as IconName } from '@rocket.chat/icons';
 import type { ReactElement, ComponentProps } from 'react';
 import React from 'react';
 
 type UserInfoActionProps = {
 	icon: IconName;
+	iconOnly?: boolean;
```

Comment:
Instead of adding a new prop, what if you just test if a label was passed with the icon? If label exists, it's a regular button, if not, IconButton.

Reply:
This one is tricky. The information passed to this component (`useUserInfoActions`) is re-used in three different places, 

- `UserCard`: shown as icon buttons 
- `UserInfo`: can be a button with label or label and icon. **Now it can also be an icon button.**
- `RoomMembers`: shows as a `GenericMenu` option

Since there many different UI's being rendered based on the same data, if I remove `content` I might break one of them. That's why I decided to add the new prop to the data and let the UI components decide how to render it.

Reply:
As we discussed in call:

Let's try to remove this change from here, and update the map in the `UserInfoActions.tsx` file to reproduce this behaviour.